AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.7
```

Comment:
Similar to smithy-swift, I don't want to change the required version yet.  Only drop to-be-unsupported Swift versions from CI for now.

Reply:
`writer.write("// swift-tools-version:${settings.swiftVersion}")`

It looks like swift-tools-version is pinned to our min swiftVersion